cves/kernel/CVE-2021-3573.yml

```diff
@@ -356,8 +366,10 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    This fix involves changing the function being used to one that did not check something 
```

Comment:
You should convert "something" to be more specific. There is also a typo in "presumably".

Reply:
thanks